front/src/applications/operationalStudies/components/SimulationResults/SpaceTimeChart/SpaceTimeChart.tsx

```diff
@@ -111,7 +147,7 @@ export default function SpaceTimeChart(props) {
    * take into account an input update
    */
   useEffect(() => {
-    setTrainSimulations(simulation.trains);
+    setTrainSimulations(simulation.trains as Train[]);
```

Comment:
It would be nice to remove all the cast in this file. I have looked a bit in the code, and I think we should update the default data to remove the cast. We will do it in a future PR.
```suggestion
    setTrainSimulations(simulation.trains);
```

Reply:
Ok, so for now, I can keep it like this ?

Reply:
I think we can keep it like this for the moment :)